web/src/emoji_picker.ts

```diff
@@ -718,6 +718,14 @@ export function toggle_emoji_popover(
         current_message_id = id;
     }
 
+    let show_as_overlay_if_reference_hidden_at_trigger = false;
+    // If the reference is in the message feed container,
+    // we want to show the popover as an overlay if the reference
+    // is hidden at the trigger time since user used a hotkey.
+    if ($(target).parents("#message_feed_container").length === 1) {
+        show_as_overlay_if_reference_hidden_at_trigger = true;
+    }
```

Comment:
I think having this check be part of the existing `popover_menus.ts` code makes more sense. Did a bit of renaming as well for clarity. I think it's probably the case we should look at other message feed popovers (user card, etc.) and see if any of them would benefit from passing this parameter.